src/cf_create.cpp

```diff
@@ -26,22 +26,17 @@ extern int get_program_skill(char_data *ch, obj_data *prog, int target);
 #define COMPLEX_FORM_TYPES 16
 
 int complex_form_programs[COMPLEX_FORM_TYPES] = {
+  SOFT_ARMOR,
   SOFT_ATTACK,
-  SOFT_SLOW,
+  SOFT_BATTLETEC,
+  SOFT_CLOAK,
+  SOFT_COMMLINK,
   SOFT_COMPRESSOR,
-  SOFT_DECRYPT,
-  SOFT_RELOCATE,
+  SOFT_LOCKON,
   SOFT_SLEAZE,
+  SOFT_SLOW,  
   SOFT_TRACK,
-  SOFT_ARMOR,
-  SOFT_CAMO,
-  SOFT_CRASH,
-  SOFT_DEFUSE,
-  SOFT_EVALUATE,
-  SOFT_VALIDATE,
-  SOFT_SWERVE,
-  SOFT_CLOAK,
-  SOFT_LOCKON
+  SOFT_SHIELD
```

Comment:
There are only 11 entries here, but 16 defined above in COMPLEX_FORM_TYPES. Does this need to be reduced?

Reply:
Good catch!